README.rst

```diff
@@ -96,7 +96,7 @@ Dependencies
 
 Basic requirements:
 
-- Python 3.6+
+- Python 3.8+
```

Comment:
We are no longer testing python 3.8, so we should best set this to 3.9+.

Reply:
that was silly